Hacktober Commit

pingrishabh opened the pull request:
UI/UX Revamp - Login & Register

-Gradient Background Added

-Title Added

-Transparent EditTexts

-EditText Drawables Added

-Green Login Button

-Carrier Buttons From Login To Register And Vice-Versa

-Screenshots Added

pingrishabh committed:
Hacktober Commit

UI/UX Revamp - Login & Register

-Gradient Background Added

-Title Added

-Transparent EditTexts

-EditText Drawables Added

-Green Login Button

-Carrier Buttons From Login To Register And Vice-Versa

-Screenshots Added

paynemiller92 commented on app/src/main/res/layout-sw320dp/activity_registration.xml:
```diff
@@ -1,90 +1,133 @@
-
-            <android.support.design.widget.TextInputLayout
+                android:layout_height="50sp"
+                android:layout_marginTop="15sp"
```
Can we please extract these hardcoded values into dimension values?

pingrishabh replied:
Moving all of these to dimens.xml

paynemiller92 commented on app/src/main/java/com/nitsilchar/hp/passwordStorage/activity/LoginActivity.java:
```diff
@@ -30,7 +30,7 @@
     String str_Password, str_getEmail, str_getPass;
     EditText edt_Email, edt_Password;
-    Button login,register;
+    Button login,register, toreg;
```
`toreg` is kind of difficult to intuitively understand. What does this button do. `login`, `register` make sense because the developer knows what they do. Any way to name `toreg` something a little more clear?

pingrishabh replied:
I used 'newuser'. That seems understandable?

paynemiller92 commented on app/src/main/java/com/nitsilchar/hp/passwordStorage/activity/LoginActivity.java:
```diff
@@ -50,25 +50,24 @@ protected void onCreate(Bundle savedInstanceState) {
         login.setOnClickListener(this);
-
-        register.setOnClickListener(new View.OnClickListener() {
+        toreg.setOnClickListener(this);
```
Why do we have two `onClickListener`s set for `toreg`? This causes unpredicatable behavior, please remove the reference to `this` and go with the anonymous inner class route.

pingrishabh replied:
Removed the extra onClickListener

paynemiller92 commented on app/src/main/res/layout-sw320dp/activity_login.xml:
```diff
@@ -4,111 +4,112 @@
+                android:fontFamily="sans-serif"
+                android:gravity="center"
+                android:background="?attr/selectableItemBackground"
+                android:text="New User? Register Here."
```
Can we please extract this into a String resource?

pingrishabh replied:
Done.

paynemiller92 commented on app/src/main/res/layout-sw320dp/activity_registration.xml:
```diff
@@ -1,90 +1,133 @@
+        android:gravity="center"
+        android:text="@string/app_name"
+        android:textColor="@android:color/white"
+        android:textSize="30sp"
```
Can we extract this into a dimension resource?

pingrishabh replied:
Done.

paynemiller92 reviewed (changes requested):
Overall, the design looks great. The code needs a bit more cleanup: clearer names, duplicate listeners, and extracting string resources.

Basically ANY xml constant: strings, colors, margins, padding, text sizes, etcetera, should be resources (string resources, dimension resources, color resources) please ensure there are no hardcoded strings within the layout xml.

Pritom14 commented:
@Cheesetouched , can you please check the reviews and put the hardcoded values into dimension values,string values. And also as pointed out by @paynemiller92 , there are multiple listeners in the activity.Please see to it that it is resolved

Pritom14 commented:
@Cheesetouched , did you commit the changes you have made?

pingrishabh commented:
Commit coming up in a moment.

pingrishabh committed:
Hacktober Commit #2

Changes Made:

-Created dimens.xml

-Removed Hardcoded String Resources

-Removed Hardcoded Dimension Resources

pingrishabh committed:
Hacktober Commit #3

-Fixed Travis CI Errors

pingrishabh commented:
The required changes were made. Travis CI pointed some reference errors. Fixed them and pushed.

Pritom14 commented:
@Cheesetouched , thanks for the PR. Can you do one more thing , now that you have changed the login as well as the registration screen can you update the login_registration screen in the same manner as you did for the two screens seperately. You can submit a new issue, I will assign that issue to you

pingrishabh commented:
Sure Thing. I'll work on it the moment I'm free.

On 28 Oct 2017 11:14 p.m., "Pritom Mazumdar" <notifications@github.com>
wrote:

> @Cheesetouched <https://github.com/cheesetouched> , thanks for the PR.
> Can you do one more thing , now that you have changed the login as well as
> the registration screen can you update the login_registration screen in the
> same manner as you did for the two screens seperately. You can submit a new
> issue, I will assign that issue to you
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Pritom14/Password-Storage/pull/31#issuecomment-340208010>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AVUJnyqaGfjqlDrrQf46D1E5-Ha4ExW7ks5sw2f5gaJpZM4QJkpI>
> .
>
